at_client/lib/src/util/at_client_validation.dart

```diff
@@ -52,4 +52,18 @@ class AtClientValidation {
       throw AtKeyException('$atSign does not exist');
     }
   }
+
+  static void validateAtKey(AtKey atKey) {
+    // validates the atKey
+    validateKey(atKey.key);
+    // validates the metadata
+    validateMetadata(atKey.metadata);
+    // verifies if the sharedWith atSign exists.
+    if (atKey.sharedWith != null) {
+      isAtSignExists(
```

Comment:
If disconnected from the network, isAtSignExists will throw an exception. In this context (validation), will the exception message make sense to a user if for example the app displays the message on screen?

Reply:
Update the code to perform this check only network connection is available.